sdk/identity/identity/src/util/tracing.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { OperationOptions } from "@azure/core-http";
 import { createSpanFunction, SpanStatusCode, Span } from "@azure/core-tracing";
+import { OperationOptions } from "@azure/core-client";
```

Comment:
Do we really need to use a specific type for options here? I think we can get away with `unknown[]` or `T[]` or something.

Reply:
I think that, if we’re not expecting these types to change drastically (and we’re not), being more specific is better!

Reply:
It ultimately should not matter but I was hoping to see if we can get rid of the dependence on `core-client` all together but it seems like you use `ServiceClient` anyway so I will leave this up to you.